crates/kubelet/src/config.rs

```diff
@@ -413,6 +424,29 @@ where
     Ok(Some(n))
 }
 
+/// Enum to choose between one wasm provider to another
+#[derive(Debug, Clone, Deserialize)]
+pub enum Provider {
+    /// Wasmtime runtime
+    WasmTime,
+    /// Wasmer runtime
+    Wasmer,
+}
+impl FromStr for Provider {
+    type Err = Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "wasmtime" => Ok(Self::WasmTime),
+            "wasmer" => Ok(Self::Wasmer),
+            other => Err(anyhow::anyhow!(
+                "unknown provider '{:?}', please choose between 'wasmer' and 'wasmtime'",
```

Comment:
There are other providers out there than just these two by the way. For example:

https://github.com/bacongobbler/krustlet-wasm3
https://github.com/kflansburg/krustlet-cri
https://github.com/wasmCloud/krustlet-wasmcloud-provider
https://github.com/deislabs/krustlet-wagi-provider
https://github.com/bacongobbler/krustlet-wasmdome
https://github.com/soenkeliebau/stackable_krustlet

These are just some of the examples I could find through Github's search engine.

I don't think we should be in the business of listing "known providers" in the crate anyways.